### PR TITLE
Change tint of progress bar

### PR DESCRIPTION
**Pull request by MillerApps**
Made the progress bar in the DApp Browser more visible as mentioned in #477.

Color: `#01CDF7`

### CODE REVIEW

**Comment by vikmeup** [comment]
@MillerApps I don't think it's a solution to the problem.

**Comment by MillerApps** [comment]
@vikmeup What did you have in mind?

**Comment by vikmeup** [comment]
How about making navigation bar white on dapps browser? and keep same progress bar? Similar to Safari?

**Comment by MillerApps** [comment]
@vikmeup Sounds good to me, I'll get it changed tonight.

File: Trust/Browser/ViewControllers/BrowserViewController.swift [hunk 1]
```diff
@@ -114,10 +114,17 @@ class BrowserViewController: UIViewController {
...
 
+        UIApplication.shared.statusBarStyle = .default
         browserNavBar?.browserDelegate = self
         reloadButtons()
     }
 
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+
+        UIApplication.shared.statusBarStyle = .lightContent
```
Comment (vikmeup) [inline Trust/Browser/ViewControllers/BrowserViewController.swift]: I don't think it's a good idea to use global status bar setting

Have you tried looking into this: https://stackoverflow.com/a/19014724?

**Comment by vikmeup** [comment]
@MillerApps can you attach a screenshot or gif as well to see how it works?

File: Trust/Browser/ViewControllers/BrowserViewController.swift (hunk 1 above)
Comment (MillerApps) [reply Trust/Browser/ViewControllers/BrowserViewController.swift]: Fixed

File: Trust/Extensions/UINavigationController.swift [hunk 2]
```diff
@@ -8,4 +8,19 @@ extension UINavigationController {
         navigationBar.tintAdjustmentMode = .normal
         navigationBar.tintAdjustmentMode = .automatic
     }
+
+    open override var preferredStatusBarStyle: UIStatusBarStyle {
+        var preferredStyle: UIStatusBarStyle
+        if navigationBar.isKind(of: BrowserNavigationBar.self) {
```
Comment (vikmeup) [inline Trust/Extensions/UINavigationController.swift]: Is there any other ways? Hardcoding this way make it hard to manage cose

Comment (vikmeup) [reply Trust/Extensions/UINavigationController.swift]: Code

Comment (MillerApps) [reply Trust/Extensions/UINavigationController.swift]: Not that I am aware of.

Comment (hewigovens) [reply Trust/Extensions/UINavigationController.swift]: usually return topViewController.preferredStatusBarStyle if you set UIViewControllerBasedStatusBarAppearance: true

Comment (MillerApps) [reply Trust/Extensions/UINavigationController.swift]: Wouldn’t the style be .default, unless each viewController provides a preferredStatusBarStyle of .lightContent?

Comment (MillerApps) [reply Trust/Extensions/UINavigationController.swift]: In order for it to work with topViewController.preferredStatusBarStyle a base UIViewController subclass would need to be used in place of UIViewController. As preferredStatusBarStyle can’t be overridden through an extension on UIViewController. So should this be the solution used?

**Pull request closed without merging**